### DISCUSSION TITLE
v2.5.3

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#147] [Announcements]
- New Italian provider GuardaHd
- Fixed Multistream and NFMirror
- Bugs Fixes and UI changes

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.5.3'>v2.5.3</a>.</em>

**Comment by IamJayDeep** [comment]
Vega team you have done great job one request i have can you add mkvcinemas as source?

**Reply by Zenda-Cross** [reply]
not possible because its has cloudflare human verification firewall
